WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift

```diff
@@ -263,4 +270,21 @@ private extension NoResultsViewController {
         return String(format: buttonIdFormat, string)
     }
 
+    // MARK: - `WPAnimatedBox` resource management
+
+    private func startAnimatingIfNeeded() {
+        guard let animatedBox = accessorySubview as? WPAnimatedBox else { return }
```

Comment:
The `return` should [be on its own line](https://github.com/wordpress-mobile/swift-style-guide#…)

Reply:
I'm happy to make the change. Can this be properly enforced with SwiftLint?